src/MoveContainer.tsx

```diff
@@ -32,16 +33,16 @@ export default class MoveContainer<
     return this._el.current!;
   }
 
-  getTemplate(): T {
-    return this._templateContainer.current!.getTemplate();
+  get template() {
+    return this._templateContainer.current!.template.current;
```

Comment:
do the .current property read inside of TemplateContainer's template getter instead of here

Reply:
[c19b6c9](https://github.com/StreakYC/react-draggable-list/pull/62/commits/c19b6c9aef71990b5bde30ad2cafb9952b197d8b)